gui/slick/views/editShow.mako

```diff
@@ -115,13 +122,6 @@ This will <b>affect the episode show search</b> on nzb and torrent provider.<br
 (check this if you wish to use the DVD order instead of the Airing order. A "Force Full Update" is necessary, and if you have existing episodes you need to move them)
 <br/><br/>
 
-% if anyQualities + bestQualities:
-<b>Archive on first match:</b>
-<input type="checkbox" name="archive_firstmatch" ${('', 'checked="checked"')[show.archive_firstmatch == 1]} /><br>
-(check this to have the episode archived after the first best match is found from your archive quality list)</br>
```

Comment:
this note is not valid, archive on first match is going to work for initial qualities. the check for any+best isn't needed, just show it as an option Always